source/includes/define-custom-user-data.rst

```diff
@@ -33,3 +33,27 @@ To enable Custom User Data in the App Services UI, follow these steps:
        leads to unexpected results.
      
 #. Save and deploy the changes.
+
+   .. note:: Restart the Sync Session after Updating Custom User Data
+
+    The sync server caches custom user data for the duration of the session.
+    As a result, you must restart the sync session after updating custom user data
+    to avoid a :ref:`compensating write error <flexible-sync-errors>`. 
+    To restart the sync session, :ref:`pause <pause-sync>` and :ref:`resume <resume-sync>` sync.
```

Comment:
I am concerned that if we link to the pause/resume docs in App Services, people may think they have to pause and resume sync on the server, and that's not the case. I'd suggest just removing those links and stating they should pause and resume sync in the client.
```suggestion
    To restart the sync session, pause and resume all open Sync Sessions in the client application.
```